INSTALL.md

```diff
@@ -0,0 +1,67 @@
+# How to install MacGitver
+
+In present state, you need to compile MacGitver from source. We provide a compilable version from our default branch `development`. If you like to help with creating binary packages, please create a [pull request](https://github.com/macgitver/MacGitver/pulls) or contact me via [mail](mailto:nils@macgitver.org).
+
+## Prerequisites
```

Comment:
No "-dev" packages needed. We already have a running dev environment by installing Qt5 via the official online installer :).

However, MGV requires installation of some `libqt5...` packages (Qt5 libs). I put a list on redmine ... let me look it up ...., yeah found it:

* libqt5core5a
* libqt5gui5
* libqt5webkit5
* libqt5widgets5

@BugSniffer This list might or might not be complete. Would be cool, if you could test that (sudo apt-get install ...).